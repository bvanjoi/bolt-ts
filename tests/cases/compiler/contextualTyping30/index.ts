// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/contextualTyping30.ts`, Apache-2.0 License

function foo(param:number[]){}; foo([1, "a"]);
//~^ ERROR: Argument of type 'number | string[]' is not assignable to parameter of type 'number[]'.