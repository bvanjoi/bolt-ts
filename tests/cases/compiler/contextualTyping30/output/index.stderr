  × Type 'string' is not assignable to type 'number'.
   ╭─[../../tests/cases/compiler/contextualTyping30/index.ts:3:41]
 3 │ "a"
   · ───
   ╰────
